dxr/build.py

```diff
@@ -487,17 +487,18 @@ def index_file(tree, tree_indexers, path, es, index, jinja_env):
     file_info = stat(path)
     folder_name, file_name = split(rel_path)
 
-    if is_text:  # conditional until we figure out how to display binary files
+    # conditional until we figure out how to display arbitrary binary files
+    if is_text or is_image(rel_path):
         es.index(index,
-                 FILE,
-                 # Hard-code the keys that are hard-coded in the browse()
-                 # controller. Merge with the pluggable ones from needles:
-                 dict(folder=folder_name,
-                      name=file_name,
-                      size=file_info.st_size,
-                      modified=datetime.fromtimestamp(file_info.st_mtime),
-                      is_folder=False,
-                      **needles))
+                FILE,
```

Comment:
The GitHub diff view makes it obvious. :-) I often catch unintended changes by doing a local git diff before I push. (And then I hook it up to FileMerge for a nice side-by-side view. GitX is pretty good, too.)=